experimental/packages/opentelemetry-instrumentation-http/src/utils.ts

```diff
@@ -348,6 +349,20 @@ export const getOutgoingRequestAttributes = (
   return Object.assign(attributes, options.hookAttributes);
 };
 
+/**
+ * Returns outgoing request Metric attributes scoped to the request data
+ * @param {SpanAttributes} spanAttributes the span attributes
+ */
+export const getOutgoingRequestMetricAttributes = (
+  spanAttributes: SpanAttributes
+): MetricAttributes => {
+  const metricAttributes: MetricAttributes = {};
+  metricAttributes[SemanticAttributes.HTTP_METHOD] = spanAttributes[SemanticAttributes.HTTP_METHOD];
+  metricAttributes[SemanticAttributes.NET_PEER_NAME] = spanAttributes[SemanticAttributes.NET_PEER_NAME];
+  metricAttributes[SemanticAttributes.HTTP_URL] = spanAttributes[SemanticAttributes.HTTP_URL];
```

Comment:
HTTP_URL in span attributes now is just the url of the request, which is a high cardinality value for metrics. I'd suggest mark this as a TODO until we have a solution to parameterize this value, as suggested in https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/metrics/semantic_conventions/http-metrics.md#parameterized-attributes.

Reply:
nvm. I find the discussion above on this.